wsi/drm/drmplane.cpp

```diff
@@ -283,8 +283,9 @@ bool DrmPlane::UpdateProperties(drmModeAtomicReqPtr property_set,
     fence = layer->GetAcquireFence();
   }
 
+   // i915 driver reads high 8bit of 16bit value
   if (layer->GetBlending() == HWCBlending::kBlendingPremult)
-    alpha = layer->GetAlpha();
+    alpha = layer->GetAlpha() << 8;
```

Comment:
Is it possible to out of scope here

Reply:
GetAlpha returns 8bit. so it is safe to left shift 8 bit

Reply:
it is not safe, alpha of layer is in type uint8_t

Reply:
It doesn't hurt to AND with the masking bits.